configure.ac

```diff
@@ -6,7 +6,7 @@ m4_define([wayland_external_micro_version], [2])
 m4_define([wayland_external_version],
           [wayland_external_major_version.wayland_external_minor_version.wayland_external_micro_version])
 
-AC_INIT([wayland_external],
+AC_INIT([egl-wayland],
```

Comment:
Thank you very much for these patches.

I agree 'wayland_external' isn't a great name for this package. The 'external' bit of it was intended to relate to the [EGL External Platform interface](https://github.com/NVIDIA/eglexternalplatform) this library actually implements.

Also, I now think 'egl-wayland' wasn't the most convenient name for this project either (sorry for the bad naming choices) as it is actually an EGLStream-specific implementation.

I'm thinking that maybe we should change it to something along the lines of 'wayland-eglstream' instead.

Also, I think it'd be fine to search & replace all occurrences of 'wayland_external' by the new chosen name throughout this file.